owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1093,89 +1093,79 @@ class FileDisplayActivity : FileActivity(),
                 customDialog.setDialogButtonClickListener(object : FileAlreadyExistsDialog.DialogButtonClickListener {
 
                     override fun onKeepBothButtonClick() {
-                        if (fileOperationsViewModel.openDialogs[posDialog].isCheckBoxChecked) {
-                            repeat(data.asReversed().size) {
-                                replace.add(false)
-                                pos++
-                                if (pos == data.size) {
-                                    launchAction(uiResult.data, replace)
-                                }
-                            }
```

Comment:
Add a coma and add the parenthesis in the next line

Reply:
Done